operator/cmd/mesh/manifest-apply.go

```diff
@@ -68,6 +70,7 @@ func addManifestApplyFlags(cmd *cobra.Command, args *manifestApplyArgs) {
 	cmd.PersistentFlags().BoolVarP(&args.wait, "wait", "w", false, "Wait, if set will wait until all Pods, Services, and minimum number of Pods "+
 		"of a Deployment are in a ready state before the command exits. It will wait for a maximum duration of --readiness-timeout seconds")
 	cmd.PersistentFlags().StringArrayVarP(&args.set, "set", "s", nil, SetFlagHelpStr)
+	cmd.PersistentFlags().StringVarP(&args.installFrom, "installFrom", "d", "", InstallFromFlagHelpStr)
```

Comment:
i feel like we need some concept of path in the name. chart-path?

Reply:
also, i think we should favor clarity over brevity, since there's a letter shortcut.

Reply:
Why do we need `path`? Helm is `helm install [CHART] [flags]` which makes sense.

I didn't realize there was a short flag though so I guess that could resolve it

Reply:
it would be different if it was a positional parameter - having a flag with a confusing name is worse in this case. i can live with --charts if you find that to be clearer.